prac_08/squaring.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 Practical
+Kivy GUI program to square a number
+Lindsay Ward, IT@JCU
+Started 13/10/2015
+"""
```

Comment:
Maybe add a docstring just saying who modified it and when? Not sure if it's necessary but I did it just in case.